src/OptimalTransport.jl

```diff
@@ -186,18 +238,19 @@ The optimal transport plan `γ` is of the same size as `C` and solves
 where ``\\Omega(\\gamma) = \\sum_{i,j} \\gamma_{i,j} \\log \\gamma_{i,j}`` is the entropic
 regularization term.
 
-Every `check_marginal_step` steps a convergence check of the error of the marginal
-`μ` with absolute tolerance `tol` is performed. After `maxiter` iterations, the
+Every `check_convergence` steps a convergence check of the error of the marginal
+`μ` with absolute tolerance `atol` and relative tolerance `rtol` is performed. The default
+`rtol` depends on the types of `μ`, `ν`, and `C`. After `maxiter` iterations, the
 computation is stopped.
```

Comment:
The same as the comment above. Perhaps we could make it clearer how the `atol` and `rtol` are used.